plugins/modules/pacemaker_resource.py

```diff
@@ -0,0 +1,227 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+
+# Copyright (c) 2025, Dexter Le <dextersydney2001@gmail.com>
+# GNU General Public License v3.0+ (see LICENSES/GPL-3.0-or-later.txt or https://www.gnu.org/licenses/gpl-3.0.txt)
+# SPDX-License-Identifier: GPL-3.0-or-later
+
+from __future__ import absolute_import, division, print_function
+__metaclass__ = type
+
+DOCUMENTATION = '''
+---
+module: pacemaker_resource
+short_description: Manage pacemaker resources
+author:
+  - Dexter Le (@munchtoast)
+version_added: 10.4.0
+description:
+  - This module can manage resources in a Pacemaker cluster using the pacemaker CLI.
+extends_documentation_fragment:
+  - community.general.attributes
+attributes:
+  check_mode:
+    support: full
+  diff_mode:
+    support: none
+options:
+  state:
+    description:
+      - Indicate desired state for cluster resource.
+    choices: [ present, absent ]
+    default: present
+    type: str
+  name:
+    description:
+      - Specify the resource name to create.
+    required: true
+    type: str
+  resource_type:
+    description:
+      - Resource type to create.
+    type: dict
+    suboptions:
+      resource_name:
+        description:
+          - Specify the resource type name.
+        type: str
+      resource_standard:
+        description:
+          - Specify the resource type standard.
+        type: str
+      resource_provider:
+        description:
+          - Specify the resource type providers.
+        type: str
+  resource_option:
+    description:
+      - Specify the resource option to create.
+    type: list
+    elements: str
+    default: []
+  resource_operation:
+    description:
+      - List of operations to associate with resource.
+    type: list
+    elements: dict
+    default: []
+    suboptions:
+      operation_action:
+        description:
+          - Operation action to associate with resource.
+        type: str
+      operation_option:
+        description:
+          - Operation option to associate with action.
+        type: list
+        elements: str
+  resource_meta:
+    description:
+      - List of meta to associate with resource.
+    type: list
+    elements: str
+  resource_argument:
+    description:
+      - Actions to associate with resource.
```

Comment:
I believe this should be singular, right?
```suggestion
      - Action to associate with resource.
```
It is one action (dict), not a list of actions.